drivers/spi/spi_ll_stm32.c

```diff
@@ -1160,6 +1165,12 @@ static const struct spi_stm32_config spi_stm32_cfg_##id = {		\
 	IF_ENABLED(DT_HAS_COMPAT_STATUS_OKAY(st_stm32_spi_subghz),	\
 		(.use_subghzspi_nss =					\
 			DT_INST_PROP_OR(id, use_subghzspi_nss, false),))\
+	IF_ENABLED(DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_spi),	\
+		(.midi_clocks =					\
+			DT_INST_PROP_OR(id, midi_clock, 0),))\
```

Comment:
As you provide a default value, prop will automatically be generated with default value (0) if not filled by the user.
Then you don't need to use `DT_INST_PROP_OR` and can just use `DT_INST_PROP(id, midi_clock)`.
Same for the other prop.

Note: It should actually be the same for `use_subghzspi_nss` as boolean properties are false if not present, no need for the `_OR` variant. I'll create an issue to get a clean up done.

Reply:
i did update `DT_INST_PROP_OR`,  may i solve use_subghzspi_nss issue with another PR if you want assign to me.

Reply:
I've created https://github.com/zephyrproject-rtos/zephyr/issues/67651 to fix this globally. You're welcome to have a stab at it!